combat-trainer.lic

```diff
@@ -1881,7 +1884,7 @@ class GameState
       echo('check_charging?: done') if $debug_mode_ct
 
       cambrinth_charges(nil)
-      invoke(@cambrinth)
+      invoke(@cambrinth, @dedicated_camb_use)
```

Comment:
Hmm I think this is more complicated than just adding it to 823.  That is sending your ritual focus into invoke - I'm not sure what would happen if I try to invoke inkwell spell